docs/01-overview.md

```diff
@@ -5,8 +5,73 @@ slug: /
+- **Hadoop Compatible File System**: Provides a filesystem interface that works with existing Hadoop ecosystem applications
+- **Rich API Options**: Supports multiple client interfaces including Java API, command-line tools, and REST endpoints
+
+### Enterprise-Ready Security
+
+- **Strong Authentication**: Integrated Kerberos authentication with robust security mechanisms
+- **Fine-Grained Authorization**: Support for both native ACLs and Apache Ranger integration for centralized authorization policies
+- **Encryption**: Transparent data encryption at rest and in-flight to protect sensitive information
+
+### Robust Data Management
+
+- **Strong Consistency**: Provides strict consistency to simplify application design and ensure data integrity
+- **Metadata Management**: Efficient handling of metadata with dedicated services for high performance
+- **Snapshots**: Support for point-in-time snapshots to protect against data corruption and facilitate backups
+
+### Operational Excellence
+
+- **Fault Tolerance**: Designed for resiliency with automatic recovery mechanisms to handle failures at all levels
+- **Observability**: Comprehensive metrics, logging, and monitoring through web UI, Prometheus integration, and Grafana dashboards
+- **Replication and Erasure Coding**: Flexible data protection strategies to balance storage efficiency and durability requirements
+
+### Integration with Data Analytics Ecosystem
+
+- **Hadoop Ecosystem**: Seamless integration with Hadoop, Hive, and Spark workloads
+- **SQL Engines**: Works with SQL query engines like Hive, Impala, and Trino without modification
+- **Modern Data Formats**: Supports modern table formats like Apache Iceberg for data lake architectures
+
+## Architecture Overview
+
+Ozone has a layered architecture that separates namespace management from storage management:
+
+- **Ozone Manager (OM)**: Manages the namespace hierarchy (volumes, buckets, and keys) and handles client metadata operations
+- **Storage Container Manager (SCM)**: Manages the containers where data is stored and handles block allocation
+- **Datanodes**: Store the actual data in containers and provide read and write access
+- **Recon**: Analytics and monitoring service that provides insight into the cluster
+
+This separation allows Ozone to achieve the scale required for modern storage systems while maintaining high performance and reliability.
+
+## Storage Elements
+
+Ozone organizes storage in a three-level hierarchy:
+
+- **Volumes**: Similar to accounts, created by administrators for organizations or teams
+- **Buckets**: Created by users within volumes, similar to S3 buckets
+- **Keys**: Data objects stored inside buckets, each potentially containing multiple blocks
+
+When a client writes data, Ozone stores it on Datanodes in chunks called blocks, which are organized into containers for efficient management and replication.
```

Comment:
This seems better suited for the architecture section. You can use Ozone's namespace layout without knowing this implementation detail.
```suggestion
When a client writes data, Ozone stores it as blocks on the Datanodes, which are organized into storage containers for efficient management and replication.
```